workflow/setup_expt.py

```diff
@@ -228,7 +228,7 @@ def input_args():
     cycled.add_argument('--nens', help='number of ensemble members',
                         type=int, required=False, default=20)
     cycled.add_argument('--app', help='UFS application', type=str,
-                        choices=['ATM', 'ATMW'], required=False, default='ATM')
+                        choices=['ATM', 'ATMW', 'ATMA'], required=False, default='ATM')
```

Comment:
The UFS-weather-model App is `ATMAERO`.
For some reason, we keep reinventing conventions.  
Is it a huge deal to now have to revert to the App defined in the model @WalterKolczynski-NOAA?

Reply:
It would make the checking slightly more complicated, because right now we just look at the last character, but easy to change. We really only use the app to set all the DO switches in `config.base`, which is what we actually use throughout. I think the reason I used a different one is I thought UFS was being inconsistent here.

Reply:
Understood.
We should follow whatever the UFS defines.  If it is inconsistent in the UFS, may be we talk to them to make it consistent.